backend/src/API/mailAPI.ts

```diff
@@ -194,3 +195,26 @@ export const sendTECReminder = async (
   } pending this semester.\n${reminder}\nTo submit your TEC, please visit https://idol.cornelldti.org/forms/teamEventCredits.`;
   return emailMember(req, member, subject, text);
 };
+
+/**
+ * Send an email reminder to members who do not have a coffee chat blackout
+ * @param req - The request made when sending the email
+ * @param member - The member being sent the email
+ * @returns - The response body containing information of the member being sent the email
+ */
+export const sendCoffeeChatReminder = async (
+  req: Request,
+  member: IdolMember
+): Promise<AxiosResponse> => {
+  const subject = 'Coffee Chat Reminder';
+
+  const isNewbie = await checkMemberMeetsCategory(member.email, member.email, 'a newbie').then(
+    (result) => result.status === 'pass'
+  );
+
+  const text = `Hey! You currently have no coffee chat bingo this semester
```

Comment:
It turns out the way that you format the message in this string is exactly how it will appear in the reminder email. I just tested it out in the deploy preview.
<img width="600" alt="Screenshot 2024-12-01 at 4 48 08 PM" src="https://github.com/user-attachments/assets/0c719cbf-bba0-4e4d-babd-f1ff720eee40">

Reply:
Also, I'm checking with Kevin about when the deadline should be. I think instead of end of semester we might want to set the deadline as last day of classes.